src/shared/components/case-viewer/case-viewer.component.html

```diff
@@ -62,7 +67,7 @@ <h2 class="heading-h2 error-summary-heading" id="edit-case-event_error-summary-h
                         <td>
                           <span class="text-16">
                             <ccd-field-read [topLevelFormGroup]="formGroup.controls['data']"
-                              [caseField]="field" [caseReference]="caseDetails.case_id">
+                                            [caseField]="field" [caseReference]="caseDetails.case_id">
```

Comment:
Minor point - What is the point of the large space that's been added in here (and below)?

Reply:
My IDE adheres to the tslint configuration hence it lined up the inputs vertical to start on the same column.

Reply:
ok, fair enough